src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -71,9 +72,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     if (comment == null) {
       response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Comment was not entered.");
       return;
+    } else {
+      long timestampMillis = System.currentTimeMillis();
+      CommentDataManager comments = new CommentDataManager();
+      comments.addItemComment(itemId, comment, timestampMillis);
     }
```

Comment:
FYI: you don't need both the early `return` and the `else` branch since they both prevent the next lines of code from getting run :)

The two options are:
```
if (comment == null) {
  response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Comment was not entered.");
} else {
  long timestampMillis = System.currentTimeMillis();
  CommentDataManager comments = new CommentDataManager();
  comments.addItemComment(itemId, comment, timestampMillis);
}
```
or
```
if (comment == null) {
  response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Comment was not entered.");
  return;
}
long timestampMillis = System.currentTimeMillis();
CommentDataManager comments = new CommentDataManager();
comments.addItemComment(itemId, comment, timestampMillis);
```